TrackletAlgorithm/ProjectionCalculator.h

```diff
@@ -237,11 +250,15 @@ template<
     nproj_disk[i] = 0;
   }
 
-  const int TCID = ((TrackletProjection<BARRELPS>::TProjTCID(Seed) << 4) + iTC);
-  const uint32_t MaskBarrel = TPROJMaskBarrel<Seed, iTC>();
-  const uint32_t MaskDisk = TPROJMaskDisk<Seed, iTC>();
+  //iTC gives you the first TCID in e.g. ABC, ipage gives you corrected TCID for B and C
+  const int TCID = ((TrackletProjection<BARRELPS>::TProjTCID(Seed) << 4) + iTC + ipage);
```

Comment:
@aryd I realize you didn't add the magic 4 here, but we can get rid of it as well:
```suggestion
  const int TCID = ((TrackletProjection<BARRELPS>::TProjTCID(Seed) << TrackletProjection<BARRELPS>::kTProjITCSize) + iTC + ipage);
```

Reply:
Done.